weave-js/src/common/util/markdown.ts

```diff
@@ -20,7 +20,12 @@ import {blankifyLinks, shiftHeadings} from './html';
 
 // exported only for tests
 export const DEFAULT_SANITIZATION_SCHEMA = _.merge(gh, {
-  attributes: {'*': ['className', 'style']},
+  attributes: {
+    '*': ['className', 'style'],
+  },
+  protocols: {
+    src: ['http', 'https', 'data'],
+  },
```

Comment:
This also has a pretty wide range of places it applies to - it’s used in the Markdown component that is used in 23 different places. One way we reduce the splash damage of this type of change is by requiring a caller to specify that we want to allow a specific change in security checks.

Reply:
➕

Reply:
I had similar concerns about the XSS potential of this change. I tried testing using usual XSS scripts both in [HTML](https://owasp.org/www-community/attacks/xss/) and [Markdown](https://book.hacktricks.xyz/pentesting-web/xss-cross-site-scripting/xss-in-markdown), but couldn't definitively say that I'm confident this is safe. Given the fact this is used across the platform, the tests aren't there either. 

Let me see who can I talk to in order to get a better sense of the implications of this change.